css/style.css

```diff
@@ -0,0 +1,56 @@
+
+html{
+  text-align: center;
+}
+.header{
+  text-align: left;
+  word-spacing: 140px;
+}
+.banner{
+  width:100%;
+  height: 80px;
+  background-image: url('../images/banner.png');
+}
+.main{
+  margin: auto 12%;
+  width: 80%
+}
+aside{
+  float:right;
+  width:20%;
+  height:100%;
+}
+
+article{
+  float:left;
+  width:80%;
+}
+.circularImage img{
+  margin: none;
+  padding: none;
+  border-radius: 50%;
+}
+
+footer{
+  border: solid black 1px;
+  clear:both;
+}
+
+@media screen and (max-width:768px){
+  html{
```

Comment:
I assume this was originally a test to check that your media query was working properly. Make sure you take out tests like this before you submit your finished product!